src/main/java/blackjack/domain/Players.java

```diff
@@ -0,0 +1,84 @@
+package blackjack.domain;
+
+import blackjack.dto.CardInfo;
+import blackjack.dto.NameInfo;
+import blackjack.dto.ScoreInfo;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Players {
+
+    private final List<Player> players;
+
+    private int activePlayerIndex;
+
+    public Players(final List<Player> players) {
+        this.players = players;
+        this.activePlayerIndex = 0;
+    }
+
+    public static Players create(final List<String> names) {
+        return new Players(
+            names.stream()
+                .map(name -> new Player(name, new Cards(new ArrayList<>())))
+                .collect(Collectors.toList())
```

Comment:
하나의 라인으로 모든 작업을 처리하려하면 오히려 가독성이 떨어질 수 있어요! 여러개의 라인으로 분리해보면 어떨까요?
```suggestion
        final List<Player> players = names.stream()
            .map(name -> new Player(name, new Cards(new ArrayList<>())))
            .collect(Collectors.toList());
        return new Players(players);
```